proposals/pattern-variables.md

```diff
@@ -101,4 +109,4 @@ These rules cover the existing top-level `is not` pattern variables. However, in
 
 ## Unresolved questions
 - Would it be possible to permit different types for each variable especially within `if`/`else` chains?
-  And if so, would it be a part of this proposal or a separate feature?
+  And if so, would it be a part of this proposal or a separate feature? (discussed in LDM 2022/10/17)
```

Comment:
No notes on that discussion?

TBH I don't know what scenario is targeted here if it's limited to identical types.

Reply:
Notes were published yesterday.

Reply:
I'm talking about if/else chains (edited my comment to clarify).